state_machine/drone.py

```diff
@@ -2,6 +2,7 @@
 
 import asyncio
 import dronekit
+import dronekit_fixes  # pylint: disable=wrong-import-order, unused-import
```

Comment:
The dronekit_fixes module no longer exists, since I have updated the flight installation guide to include the DroneKit wheel file.